llvm/lib/YkIR/YkIRWriter.cpp

```diff
@@ -963,6 +964,39 @@ class YkIRWriter {
     InstIdx++;
   }
 
+  /// Serialise a cast-like instruction.
```

Comment:
Kill/update this comment.

Reply:
A truc is still a cast, but I can update it.